examples/bert/run_glue_finetuning.py

```diff
@@ -64,14 +65,11 @@
     "Whether to run evaluation on the validation set for a given task.",
 )
 
-flags.DEFINE_integer("batch_size", 32, "The batch size.")
-
-flags.DEFINE_integer("epochs", 3, "The number of training epochs.")
-
-flags.DEFINE_float("learning_rate", 2e-5, "The initial learning rate for Adam.")
-
 flags.DEFINE_integer("max_seq_length", 128, "Maximum sequence length.")
 
+batch_size = 32
```

Comment:
We should keep the flag with proper default value instead of making it a constant.

Reply:
@mattdangerw Matt do we want to always have the same value here? If so, we can stick to a constant, in which case we might rename it to capital case?

Reply:
The paper suggests 32 batch size for all glue finetuning tasts, so I think this is fine.

I think we will likely want to some more adjustments of how we run anyway (e.g. maybe push more train parameters into the json config files), so I don't think we need to solve this on this PR.

Reply:
Style nit though, constants should be ALL_CAPS.